Assets/Scripts/MainMenu.cs

```diff
@@ -0,0 +1,51 @@
+using UnityEngine;
+using UnityEngine.Audio;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+
+public class MainMenu : MonoBehaviour
+{
+    [SerializeField] private AudioMixer audioMixer;
+    [SerializeField] private Slider BGMSlider;
+    [SerializeField] private Slider SFXSlider;
+    [SerializeField] private float startBGMVolume = 0.5f;
+    [SerializeField] private float startSFXVolume = 0.5f;
+    [SerializeField] private float startMasterVolume = 0.5f;
+    [SerializeField] private string startingScene;
+
+    public void Start()
+    {
+        audioMixer.SetFloat("Master", startMasterVolume);
+        var savedBGMVolume = PlayerPrefs.GetFloat("BGM", startBGMVolume);
+        var savedSFXVolume = PlayerPrefs.GetFloat("SFX", startSFXVolume);
+        SetBGMVolume(savedBGMVolume);
+        SetSFXVolume(savedSFXVolume);
+        BGMSlider.value = savedBGMVolume;
+        SFXSlider.value = savedSFXVolume;
+    }
+
+    public void SetBGMVolume(float volume)
```

Comment:
Public functions like these should have descriptive comments (eg. if I want to change the volume, what range of float values should I use?)